spanner/session.go

```diff
@@ -703,6 +706,7 @@ func newSessionPool(sc *sessionClient, config SessionPoolConfig) (*sessionPool,
 	if isMultiplexed != "" && isMultiplexed != "true" && isMultiplexed != "false" {
 		return nil, spannerErrorf(codes.InvalidArgument, "GOOGLE_CLOUD_SPANNER_MULTIPLEXED_SESSIONS must be either true or false")
 	}
```

Comment:
I would prefer doing this env detection for read-write here instead of having it in client.go file.
Do you see any issues with that?